biotrainer/embedders/embedding_service.py

```diff
@@ -230,6 +232,42 @@ def _save_and_reset_embeddings(self, embeddings: Dict[str, ndarray], last_save_i
         del embeddings
         return last_save_id, {}
 
+    @staticmethod
+    def reduce_embeddings_dimension(
+        embeddings: Dict[str, Any],
+        dimension_reduction_method: str,
+        reduced_dimension: int) -> Dict[str, Any]:
+        """
+        Reduces the dimension of per-residue embeddings or per-protein embeddings
+
+        Parameters:
+            embeddings (Dict[str, ndarray]): Dictionary of embeddings.
+            embedder: The embedder used for reducing embeddings.
```

Comment:
Please add the missing function parameters here (and remove embedder)

Reply:
sorry I copied this from the _reduce_embeddings function and forgot to change it